node/server.ts

```diff
@@ -94,6 +102,38 @@ function startTheServer(pathToServe: string, port: number): void {
   //   res.end();
   // });
 
+
+  app.get('/video', (req, res) => {
+    const seekTime = req.query.seek || 0;
+    const file = req.query.file || '';
+    // see https://trac.ffmpeg.org/wiki/Encode/H.264#a2.Chooseapreset for more options
+    const ffmpeg = spawn(ffmpegPath, [
+      '-ss', seekTime,
+      '-i', file,
+      '-f', 'mp4',
+      '-crf', '17',
+      '-preset', 'ultrafast',
+      '-movflags', 'frag_keyframe+empty_moov+faststart',
+      '-frag_duration', '15',
```

Comment:
You can probably remove this - I think it might have a sensible default anyway. We can change it later.

The idea with fragmenting, is that you can seek and stream much easier, without having to download the entire mp4 file before playback can begin. Each fragment contains metadata about the video, rather than all the metadata being store at the end of the file!

Reply:
I can look into finding the best options, but if the code works as is, may as well use that for now!

Reply:
Thanks Cal for looking at this again. I've paused this feature for a while because I had at least two days of trying hard with no success: getting the video to play in Chrome ✅ no problem 👌 but getting Safari on iOS (on my phone) to play anything was a no-go 😓 which is the sole purpose of this feature.

I'll come back to this with renewed energy once I move in to the new house (these few months have been busy).

😅 also I've been working on an image-centric analogous-to-VHA app this past month -- I want to finish it first before returning to this headache :trollface: